content/index.ipynb

```diff
@@ -103,9 +103,9 @@
     "\n",
     "## 2.1 Participants\n",
     "\n",
-    "Two hundred and three participants from the _spine-generic_ dataset (multi-subject) were used {cite:p}`Cohen-Adad2021-tn`. The dataset is open-access, organized according to the Brain Imaging Data Structure (BIDS) standard {cite:p}`Gorgolewski2016-xf,Karakuzu2022-bids` and managed using [git-annex](href=\"https://git-annex.branchable.com) in [this GitHub repository](https://github.com/spine-generic/data-multi-subject/tree/r20230223).\n",
+    "Two hundred and three participants from the _spine-generic_ dataset (multi-subject) were used {cite:p}`Cohen2021-tn`. The dataset is open-access, organized according to the Brain Imaging Data Structure (BIDS) standard {cite:p}`Gorgolewski2016-xf,Karakuzu2022-bids` and managed using [git-annex](href=\"https://git-annex.branchable.com) in [this GitHub repository](https://github.com/spine-generic/data-multi-subject/tree/r20230223).\n",
```

Comment:
Maybe `CohenAdad2021` would be better `Cohen2021`?

Reply:
These keys don't matter much, they are not rendered in the end.